docs/sources/flow/reference/components/prometheus.exporter.squid.md

```diff
@@ -0,0 +1,100 @@
+---
+# NOTE(rfratto): the title below has zero-width spaces injected into it to
+# prevent it from overflowing the sidebar on the rendered site. Be careful when
+# modifying this section to retain the spaces.
+#
+# Ideally, in the future, we can fix the overflow issue with css rather than
+# injecting special characters.
+
+title: prometheus.exporter.squid
+---
+
+# prometheus.exporter.squid
+The `prometheus.exporter.squid` component embeds
+[squid_exporter](https://github.com/grafana/squid-prometheus-exporter) for collecting warehouse, database, table, and replication statistics from a squid account via HTTP for Prometheus consumption.
+
+## Usage
+
+```river
+prometheus.exporter.squid "LABEL" {
+    address = "<HOST>:<PORT>"
```

Comment:
Could you align all the strings in the same column? Ideally you could use `riverfmt` to format your river code. Same for the examples in the test.